lib/elixir/lib/exception.ex

```diff
@@ -775,6 +775,25 @@ defmodule UnicodeConversionError do
   end
 end
 
+defmodule Process.RegisterError do
+
+  def message(pid, name, original_message) do
```

Comment:
Since we never raise this. I would move this function directly to the Process module and keep it private.